src/eos-metrics-instrumentation.c

```diff
@@ -67,6 +67,18 @@
  */
 #define NETWORK_STATUS_CHANGED_EVENT "5fae6179-e108-4962-83be-c909259c0584"
 
+/*
+ * Recorded at every startup to track deployment statistics. The auxiliary
+ * payload is a 3-tuple of the form (os_name, os_version, eos_personality).
+ */
+#define OS_VERSION "1fa16a31-9225-467e-8502-e31806e9b4eb"
+
+#define OSRELEASE_FILE "/etc/os-release"
+
+#define PERSONALITY_FILE_PATH "/etc/EndlessOS/personality.conf"
```

Comment:
It should exist on all images. There was a discussion about "personality-less" images, but I don't remember what the resolution for that was.